ffi_internals/src/struct_internals/struct_ffi.rs

```diff
@@ -45,21 +45,21 @@ impl StructFFI {
     ///
     #[must_use]
     pub fn init_fn_name(&self) -> Ident {
-        format_ident!("{}_init", self.name.to_string().to_snake_case())
+        format_ident!("{}_rust_ffi_init", self.name.to_string().to_snake_case())
```

Comment:
Changing these updates the `TokenStream` that produces the FFI module for each struct, and the `ConsumerStruct` that produces Swift code for calling it, so it takes care of both sides at once.